common/src/main/java/com/wynntils/mc/event/ChatSendMessageEvent.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.mc.event;
+
+import net.minecraftforge.eventbus.api.Event;
+
+public class ChatSendMessageEvent extends Event {
+
+    private final String message;
+
+    public ChatSendMessageEvent(String message) {
+        this.message = message;
+    }
+
+    public String getMessage() {
+        return message;
+    }
+
+    @Override
+    public boolean isCancelable() {
```

Comment:
you can use the `@Cancelable` annotation instead of including this override

Reply:
Didn't know that, thanks, will fix.